lib/pure/distros.nim

```diff
@@ -201,7 +198,7 @@ proc detectOsImpl(d: Distribution): bool =
 
 template detectOs*(d: untyped): bool =
   ## Distro/OS detection. For convenience the
-  ## required ``Distribution.`` qualifier is added to the
+  ## required `Distribution.` qualifier is added to the
```

Comment:
since this relies on nimlint automation, why not try it on a larger set of files, eg `lib/pure/*` or even `lib/*`, and, say, just do it for 1 linter check (double backticks => single backticks) (single lint category makes reviewing easier)